nuxeo-data-table/data-table-row.html

```diff
@@ -104,6 +104,13 @@
               type: Object,
               value: { id: 0 },
             },
+            /**
+             * Extended Nuxeo-Data-Table tag name
+             */
+            dataTableTagName: {
```

Comment:
We'd likely not expose this as a public property in any case, just `_tagName` would be better.

Reply:
done

Reply:
renamed to _hostTagName to reflect the purpose of this property and be "private"